run/test/specs/onboarding_tos.spec.ts

```diff
@@ -30,13 +28,11 @@ async function onboardingTOS(platform: SupportedPlatformsType) {
     const retrievedURL = await device.getTextFromElement(urlField)
     // Add https:// to the retrieved URL if the UI doesn't show it (Chrome doesn't, Safari does)
     const fullRetrievedURL = retrievedURL.startsWith('https://') ? retrievedURL : `https://${retrievedURL}`;
-    console.log(`The URL is "${fullRetrievedURL}"`);
-    console.log(`The expected URL is "${tosURL}"`);
     // Verify that it's the correct URL 
-    if (fullRetrievedURL === tosURL) {
-        console.log("The URLs match.");
-    } else {
+    if (fullRetrievedURL !== tosURL) {
         throw new Error("The retrieved URL does not match the expected.");
```

Comment:
Yeah add in the URL that is returned here

Reply:
done